cookbook/product/query-products.rst

```diff
@@ -59,16 +59,80 @@ Add sorters:
         // sort by completeness, the locale and scope is expected, if not provided, the default one are used
         ->addSorter('completeness', 'DESC', ['locale' => 'fr_FR', 'scope' => 'mobile']);
 
-Execute the query
+Execute the Query to Get a Cursor
+---------------------------------
+
+It will return a `Akeneo\Component\StorageUtils\CursorInterface` on the products collection, this interface is "storage agnostic".
```

Comment:
would be good to explain the advantage of the cursor: storage agnositc, memory friend etc..